README.md

```diff
@@ -41,11 +41,15 @@ docker compose up
 
 In order to user Docker on Windows you may install [Docker desktop for Windows](https://docs.docker.com/desktop/windows/install/)
 
-You can execute same commands as Linux part given above.
 
-Or in command powershell execute:
+If you have some distribution installed (Ubuntu, ...) in WSL, you can simply run the linux command as above, once in it. 
 
-```
+Or in powershell, run the folloinwg command to setup some files
```

Comment:
```suggestion
Or in PowerShell, run the following command to set up some files
```

* Setup is a noun, for the verb, it's two words : https://grammarist.com/spelling/set-up-vs-setup/
* Maybe a add a quick sentence that the QGIS server version is hard-coded ?